dotcom-rendering/src/components/MostViewedFooterItem.tsx

```diff
@@ -2,49 +2,31 @@ import type { SerializedStyles } from '@emotion/react';
 import { css } from '@emotion/react';
 import type { ArticleFormat } from '@guardian/libs';
 import { ArticleDesign } from '@guardian/libs';
-import { headlineMedium17, until } from '@guardian/source/foundations';
+import { headlineMedium17 } from '@guardian/source/foundations';
 import { palette } from '../palette';
 import { AgeWarning } from './AgeWarning';
 import { BigNumber } from './BigNumber';
 import { FormatBoundary } from './FormatBoundary';
 import { LinkHeadline } from './LinkHeadline';
 
-const gridItem = (position: number, hasPageSkin: boolean) => {
-	const borderTop = hasPageSkin
-		? css`
-				border-top: 1px solid ${palette('--article-border')};
-		  `
-		: css`
-				/* Below leftCol always set top border */
-				${until.leftCol} {
```

Comment:
From leftCol, there is a double border below the heading

<img width="600" alt="Screenshot 2024-07-22 at 09 18 57" src="https://github.com/user-attachments/assets/c7f8480f-6d78-4cd8-b578-8ed090653c48">